cmd/pebble/cmd_run.go

```diff
@@ -180,6 +197,10 @@ out:
 	for {
 		select {
 		case sig := <-ch:
+			if sig == syscall.SIGTERM && rcmd.SigtermAction == sigtermActionIgnore {
```

Comment:
This will probably need more in the future (if we want to respond to other signals) but this is ok for now.